common/tools/dev-tool/src/util/customization/helpers/annotations.ts

```diff
@@ -14,12 +19,14 @@ export function getAnnotation(
     for (const commentRange of leadingCommentRanges) {
       const commentText = commentRange.getText();
 
-      const regex = /@azsdk-(\w+)/;
+      const regex = /@azsdk-(\w+)(?:\((\w+)\))?/;
```

Comment:
This is fine for now, but moving forward, we'll have to consider how we want to go about parsing these annotations. At the moment, we don't have any validation for arity or type-correctness in the annotation parameters.

Reply:
yeah, this is a very simple addition and assumes the user will do the right thing.